controllers/player.controller.js

```diff
@@ -1,43 +1,90 @@
 const Player = require('../models/player.model')
 
+const logError = requestId => {
```

Comment:
i would put this in a helper file

Reply:
you should also expect a message not only `requestId` and pass the message from everywhere